scripts/snp_phylogenomics_control.pl

```diff
@@ -717,8 +717,6 @@ =head2 MAPPING
 
 =item B<pseudoalign>:  Generates pseudoalignment file.
 
-=item B<vcf2core>:  Generates gview file(s)
```

Comment:
Could you actually re-add this comment, but change to **Computes percent of genome used for analysis.**?